buildroot-external/rootfs-overlay/usr/libexec/hassos-zram

```diff
@@ -57,7 +57,7 @@ if [ "$TYPE" = "swap" ] && [ "$SIZE" -eq "0" ]; then
 fi
 
 # Init device
-zramctl "$DEVICE" -s "$SIZE" -a lz4
+zramctl "$DEVICE" -s "$SIZE"
```

Comment:
zram has new config options to specify which compression algorithms are included. It defaults to `lzo` unless `lz4` is explicitly enabled. It looks like lz4 was set 5+ years ago--not sure if it was intentionally to use this algorithm or an outdated default

However, I also enabled `lz4` compression in the kernel config options. I don't think both changes are needed--not sure best path
- use default zram compression
- keep as existing lz4

Reply:
It looks like maybe lz4 is a better option since it's lower CPU usage and lzo-rle (default) doesn't have much better compression

ref https://www.reddit.com/r/Fedora/comments/mzun99/new_zram_tuning_benchmarks/